bears/php/PHPStanBear.py

```diff
@@ -0,0 +1,21 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='phpstan',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)  (?P<message>.*)')
+class PHPStanBear:
+    """
+    Checks the code with ``phpstan analyze``.
+    This can run it on multiple files and folders.
+    See <https://github.com/phpstan/phpstan> for more information.
+    """
+    LANGUAGES = {'PHP'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax'}
```

Comment:
Yup added the following`CAN_DETECT = {'Syntax','Unused Code','Variable Misuse','Undefined Element','Missing Import','Spelling'}`
Will be updating soon.